sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchRequest.java

```diff
@@ -223,6 +223,11 @@ public final class SearchRequest implements JsonSerializable<SearchRequest> {
      */
     private VectorFilterMode vectorFilterMode;
 
+    /*
+     * The query parameters to configure hybrid search behaviors.
+     */
+    private Object hybridSearch;
```

Comment:
Seems the Swagger is slightly off here, this should have been the type `HybridSearch`

Reply:
I'll add a customization for this